recipes/manhole/meta.yaml

```diff
@@ -27,8 +27,8 @@ requirements:
 test:
   imports:
     - manhole
-  commands:
-    - manhole-cli --help
+  commands:  # [not win]
+    - manhole-cli --help  # [not win]
```

Comment:
Over DM I suggested making this recipe architecture-dependent, as the package depends on `readline` that does not exist for Windows. Then, you won't need to add these selectors.